plugin/pkg/admission/imagepolicy/admission_test.go

```diff
@@ -211,7 +213,7 @@ current-context: default
 					return fmt.Errorf("failed to execute test template: %v", err)
 				}
 
-				tempconfigfile, err := ioutil.TempFile("", "")
+				tempconfigfile, err := os.CreateTemp("", "")
```

Comment:
Also need to defer close here?

Reply:
This whole function is fishy - below we try to Open(pc) - it's already open!

Reply:
This function seems to be fine, configTmpl.Execute() will make the file pointer at the end of the file, if the next time the file is read, the read operation will start from the end of the file, so we need to re-open it